deploy/helm/values.yaml

```diff
@@ -0,0 +1,91 @@
+# Default values for starboard-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# targetNamespace defines where you want starboard-operator to operate. By
+# default it will only operate in the namespace its installed in, but you can
+# specify another namespace, or a comma separated list of namespaces, or set it
+# to a blank string to let it operate in all namespaces.
+targetNamespaces: "{{ .Release.Namespace }}"
+
+image:
+  repository: "docker.io/aquasec/starboard-operator"
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "dev"
+  pullPolicy: ""
+  pullSecrets: []
+
+replicaCount: 1
+
+nameOverride: ""
+fullnameOverride: ""
+
+# envSecret represent a k8s Secret resource that will be referenced to mount
+# environment variables.
+envSecret:
+  # create specifies whether a k8s Secret should be created with a given
+  # stringData. If this is set to false, only the keys defined in stringData
+  # will be optionally mounted from the named secret, rather than all keys in
+  # the named k8s Secret.
+  create: true
+  # name specifies the name of the k8s Secret to reference. If not set, a name
+  # is generated using the fullname template.
+  name: ""
+  # stringData specifies key value pairs to be added to the k8s Secret and
+  # mounted by the Pod as environment variables.
+  #
+  # NOTE: It is important that you specify targetNamespace instead of
+  #       OPERATOR_TARGET_NAMESPACE below for this Helm chart to function
+  #       properly.
+  stringData:
+    OPERATOR_LOG_DEV_MODE: "false"
+    OPERATOR_SCANNER_TRIVY_ENABLED: "true"
+    OPERATOR_SCANNER_TRIVY_VERSION: "0.11.0"
+    OPERATOR_SCANNER_AQUA_CSP_ENABLED: "false"
+    OPERATOR_SCANNER_AQUA_CSP_VERSION: ""
+    OPERATOR_SCANNER_AQUA_CSP_HOST: ""
+    OPERATOR_SCANNER_AQUA_CSP_USER: ""
+    OPERATOR_SCANNER_AQUA_CSP_PASSWORD: ""
+
+rbac:
+  create: true
+serviceAccount:
+  # Specifies whether a service account should be created.
+  create: true
+  annotations: {}
+  # name specifies the name of the k8s Service Account. If not set and create is
+  # true, a name is generated using the fullname template.
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  runAsNonRoot: true
+  allowPrivilegeEscalation: false
+  runAsUser: 65534
+  runAsGroup: 65534
+  readOnlyRootFilesystem: true
+  capabilities:
+    drop:
+      - ALL
```

Comment:
I've opted to drop all privileges and planned to add more if needed, perhaps I need to be able to do something in order to expose the metrics for example.